src/components/DelegateStatement/DelegateStatementForm.tsx

```diff
@@ -64,7 +62,7 @@ export default function DelegateStatementForm({
     if (!walletClient) {
       throw new Error("signer not available");
     }
-    values.topIssues = cleanTopIssues(values.topIssues);
+    values.topIssues = values.topIssues.filter((issue) => issue.value !== "");
```

Comment:
similar to feedback you gave me before about mutating the object -- I don't think it would effect behavior but could probably do something like

```
const updatedValues = {
...values,
topIssues: values.topIssues.filter((issue) => issue.value !== "")
}
```

then use updatedValues in place of values